src/hidpp20.h

```diff
@@ -787,7 +787,13 @@ hidpp20_onboard_profiles_write_sector(struct hidpp20_device *device,
 static inline uint8_t *
 hidpp20_onboard_profiles_allocate_sector(struct hidpp20_profiles *profiles)
 {
+#ifndef __clang_analyzer__
 	return zalloc(profiles->sector_size);
+#else
+	/* clang static analyzer doesn't know how to handle _cleanup_
+	 * https://bugs.llvm.org/show_bug.cgi?id=3888 */
+	return 1;
+#endif
```

Comment:
I'd probably go with 
```
#ifdef __clang_analyzer__
return 1
#endif
```
instead to reduce the cognitive load on the normal branch. Even though it's just a single line here.